src/commands/publish.rs

```diff
@@ -51,7 +57,21 @@ pub fn publish(
             delete_bulk(target, user, &site_namespace.id, to_delete)?;
         }
     } else {
-        upload::script(&user, &target, None)?;
+        let uses_kv_bucket = sync_non_site_buckets(target, user, verbose)?;
+
+        let feature = if uses_kv_bucket {
+            let wrangler_toml = style("`wrangler.toml`").yellow().bold();
+            let issue_link = style("https://github.com/cloudflare/wrangler/issues/new?labels=user+report&template=feature_request.md").blue().bold();
```

Comment:
actually after reading @EverlastingBugstopper 's pr, i want to make this a specific issue, probably called "deprecate kv_namespaces.bucket" or something.